go.mod

```diff
@@ -3,26 +3,16 @@ module map
 go 1.12
 
 require (
-	github.com/Workiva/go-datastructures v1.0.50 // indirect
-	github.com/emirpasic/gods v1.12.0 // indirect
-	github.com/ethereum/go-ethereum v1.8.27 // indirect
+	github.com/blang/semver v3.5.1+incompatible
+	github.com/ethereum/go-ethereum v1.9.13
 	github.com/gin-contrib/cors v1.3.0
 	github.com/gin-gonic/gin v1.4.0
 	github.com/go-sql-driver/mysql v1.4.1
-	github.com/gogo/protobuf v1.2.1 // indirect
-	github.com/gorilla/websocket v1.4.0 // indirect
-	github.com/gosuri/uilive v0.0.3 // indirect
-	github.com/gosuri/uiprogress v0.0.1 // indirect
-	github.com/hashicorp/golang-lru v0.5.1
-	github.com/howeyc/gopass v0.0.0-20170109162249-bf9dde6d0d2c // indirect
+	github.com/hashicorp/golang-lru v0.5.3
 	github.com/imroc/req v0.2.3
-	github.com/itchyny/base58-go v0.0.5 // indirect
-	github.com/ontio/ontology v1.6.2
-	github.com/ontio/ontology-crypto v1.0.4 // indirect
+	github.com/ontio/ontology v0.0.0-20200624063426-fcbf82cb6203
```

Comment:
anchor to v2.0.0 ?

Reply:
ontology version above 2.0, marjor version here must be compatible or v0.0.0-20200624063426-fcbf82cb6203